### DISCUSSION TITLE
Argos Translate in Arch User Repositories (not by me)

### DISCUSSION THREAD

**Discussion by PJ-Finlay** [argosopentech/argos-translate#79] [General]
[
![image](https://user-images.githubusercontent.com/13067016/114248733-11d3c980-995e-11eb-8d91-4da9305bc47a.png)
](https://aur.archlinux.org/packages/argos-translate-git/)

**Comment by PJ-Finlay** [comment]
![image](https://user-images.githubusercontent.com/13067016/114248769-329c1f00-995e-11eb-9449-684d3c445485.png)

There's really no reason to install as root but I'm sure they can figure it out. [Dependency management is currently being worked on](https://github.com/argosopentech/argos-translate/issues/75#issuecomment-815721035) but installing with pip is still probably the best option.

**Reply by sandsmark** [reply]
Yeah, the reason I was surprised was that it abused the post-install scripting instead of actually trying to package it properly. If I wanted it installed system-wide with pip I could just run pip.

It's fairly easy to fix, just add the dependencies as dependencies in the PKGBUILD, I'm fairly sure all of them are packaged and available already.
